legate/core/operation.py

```diff
@@ -359,36 +404,55 @@ def get_requirement(store, part_symb):
 
 
 class ManualTask(Task):
-    def __init__(self, context, task_id, launch_domain, mapper_id=0, op_id=0):
-        Task.__init__(self, context, task_id, mapper_id, op_id)
-        self._launch_domain = launch_domain
-        self._input_projs = []
-        self._output_projs = []
-        self._reduction_projs = []
+    def __init__(
+        self,
+        context: Context,
+        task_id: int,
+        launch_domain: Rect,
+        mapper_id: int = 0,
+        op_id: int = 0,
+    ) -> None:
+        super().__init__(context, task_id, mapper_id, op_id)
+        self._launch_domain: Rect = launch_domain
+        self._input_projs: list[Union[ProjFunc, None]] = []
+        self._output_projs: list[Union[ProjFunc, None]] = []
+        self._reduction_projs: list[Union[ProjFunc, None]] = []
+
+        self._input_parts: list[StorePartition] = []  # type: ignore [assignment] # noqa: E501
```

Comment:
is there a reason for the comment `# type: ignore [assignment]`?

Reply:
This is due to the incompatible overrides: 
```
legate/core/operation.py:421:51: error: Incompatible types in assignment (expression has type "List[StorePartition]", base class "Operation" defined the type as "List[PartSym]")  [assignment]
```
This could/should probably be improved but for now this comment just documents what currently is

Reply:
I can make a new issue for this (here I assume, not Jira?)

Reply:
My perception is that very specific technical tasks go on GH so I have made https://github.com/nv-legate/legate.core/issues/193